app/views/bulkrax/entries/show.html.erb

```diff
@@ -34,18 +34,22 @@
     <p class='bulkrax-p-align'>
       <% if @importer.present? %>
 	    <%# TODO Consider how to account for Bulkrax.collection_model_class %>
-        <% factory_record = @entry.factory.find %>
-        <% if factory_record.present? %>
-          <% factory_record_class = factory_record.class %>
-          <% factory_record_class_human = factory_record_class.model_name.human %>
-          <strong><%= factory_record_class_human %> Link:</strong>
-          <% if defined?(Hyrax) && factory_record_class_human == 'Collection' %>
-            <%= link_to factory_record_class_human, hyrax.polymorphic_path(factory_record) %>
+        <% begin %>
+          <% factory_record = @entry.factory.find rescue nil %>
```

Comment:
Now that find works, we need a way to handle cases where the record isn't found (when we successfully delete a record, it is no longer there and we don't want the find error to bubble up here)